src/Raven.Server/Documents/Indexes/Persistence/Lucene/IndexReadOperation.cs

```diff
@@ -847,6 +860,15 @@ public override void Dispose()
             _analyzer?.Dispose();
             _releaseSearcher?.Dispose();
             _releaseReadTransaction?.Dispose();
+
+            if (_memoryInfo != null && _memoryInfo.ManagedThreadId == NativeMemory.CurrentThreadStats.ManagedThreadId)
+            {
+                var diff = GC.GetAllocatedBytesForCurrentThread() - _memoryInfo.AllocatedBefore;
+                if (diff > 0)
```

Comment:
This is always enabled?

Reply:
Only when the information Logs are enabled.
https://github.com/ravendb/ravendb/pull/14749/files#diff-06a440595965380fa42c2760f65cc3241ac73b52ff20be53bb0f16eda826cc9bR85